okonomiyaki/file_formats/_egg_info.py

```diff
@@ -1015,6 +1054,28 @@ def dump(self, path):
             if self.pkg_info:
                 self.pkg_info._dump_as_zip(zp)
 
+    def to_json_dict(self):
+        if self.platform is None:
+            epd_platform = None
+        else:
+            epd_platform = six.text_type(self.platform)
+
+        return {
+            _JSON_METADATA_VERSION: six.text_type(self.metadata_version),
+            _JSON__RAW_NAME: self._raw_name,
+            _JSON_NAME: self.name,
```

Comment:
the name vs so-called raw name distinction is needed for various compatibility reasons. But here, `EggMetadata` is handling it for us, with `name` being deduced from the `raw_name`.